pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -224,6 +241,8 @@ func (m *Manager) processShutdownEvent() error {
 
 	nonCriticalPodGracePeriod := m.shutdownGracePeriodRequested - m.shutdownGracePeriodCriticalPods
 
+	m.recorder.Eventf(m.nodeRef, v1.EventTypeNormal, kubeletevents.NodeShutdown, "Shutting down %v pods", len(activePods))
+
```

Comment:
IMO this seems like a duplicate of the `Shutdown manager detected new shutdown event`, I don't think it's useful to have two events here. I would either combine with the previous event or remove this entirely.

Reply:
fixed. thanks!